model/metricset.go

```diff
@@ -132,6 +132,33 @@ func (s *SummaryMetric) fields() mapstr.M {
 	return mapstr.M{"value_count": s.Count, "sum": s.Sum}
 }
 
+// AggregateMetric holds extended summary metrics (value_count, sum, min, max).
+type AggregateMetric struct {
```

Comment:
There's an existing type [`model.SummaryMetric`](https://github.com/elastic/apm-server/blob/0420137bea7f9e8ad10ce6b5bbb751b0e0134baa/model/metricset.go#L122-L129), which is a subset of this. If Min and Max aren't needed, we should use that. If they are needed, I'd like to find a way to unify them.